### ISSUE TITLE
[BUG] - Lost toggle between God mode and 3rd person mode for creating Blueprints

### ISSUE THREAD

**Issue by Chemie99** [kremnev8/DSP-Mods#24]
**Describe the bug**
No longer able to change between the two view modes with "J" key since version  0.8.23.9925

**To Reproduce**

1. Click "scissors" to start Blueprint copy mode
Game enters GOD mode
2. Click on "J" to toggle mode; it does nothing

**Expected behavior**
"J" used to toggle between GOD and 3rd person; Now locked to God mode.

**Additional context**
J was not remapped after update.  My game does not have J mapped.  J used to work but I am now locked in GOD mode. Option is enabled in config settings.

**Comment by kremnev8** [comment]
Unfortunately, but I'm unable to reproduce this issue. `J` works for me in the latest version. Can you double check that you haven't rebound the key. It would be also helpful if you provide me with your game log. It can be found at `Dyson Sphere Program\BepInEx\LogOutput.log` (Make sure to load the game before that and try pressing the key)

**Comment by Chemie99** [comment]
> Unfortunately, but I'm unable to reproduce this issue. `J` works for me in the latest version. Can you double check that you haven't rebound the key. It would be also helpful if you provide me with your game log. It can be found at `Dyson Sphere Program\BepInEx\LogOutput.log` (Make sure to load the game before that and try pressing the key)

Thanks for the reply.  I guess it is good it is me and not the mod but not sure where the conflict is coming from.  I will say I also had "betterstats" break and someone on Reddit said it still worked for them so not sure if related or not.  We had a couple of game updates and some of the MOD tool dependencies also updated recently (CommonAPI and DSP Mod save) so my assumption was one of those broke both Blueprinttweaks and Betterstats but I guess it is on my end.  

I tried:
* checked game file integrity
* Confirmed "J" is not mapped to anything in game 
* Turned off all mods except Blueprinttweaks (leaving the dependencies enabled) and still was locked in GOD mode with trying to paste a blue print. (As an aside, Betterprints as only mod is also broken so I don't it is other mods conflicting but could be with one of the libraries.)
* Confirmed God mode not enabled in the game settings.
I do not have a Logoutput.log file in the location you gave, nor anywhere else associated with DSP game files.  I have r2modman installed standalone folder (and BepInEx is in its own subfolder there in the \mods\DysonSphereProgram\cache folder but no log file there.  Also no log files in the DSP game folder my Steam folder or the the "Mydocuments\DysonSphereprogram folder)

**Comment by Chemie99** [comment]
I noticed R2Modman had updated Nov. 18 which was right around when things broke.  So I manually added Betterstats and BPtweaks mods to DSP folders.  The Betterstats then worked (although BPtweaks did not but I suspect I missed a step somewhere perhaps with the patchers needed).
So I did full reinstall of the game and uninstall/reinstall of r2modman.  This then fixed BlueprintTweaks!
So it appears the R2modman patch messed something up (I am still running the newest version; I think it was related to the profiles but did not fully diagnose)
